packages/storybook-html/src/components/core/colour-apply.story.mdx

```diff
@@ -274,30 +296,35 @@ These can be used on a white background or as background colours to white text,
     <th class="uqds-table__heading">Example</th>
     <th class="uqds-table__heading">Name</th>
     <th class="uqds-table__heading">Variable</th>
+    <th class="uqds-table__heading">HEX</th>
     <th class="uqds-table__heading">Usage</th>
   </tr>  
   <tr  class="uqds-table__row">
     <td class="uqds-table__item" data-title="Example:">Not yet defined</td>
     <td class="uqds-table__item" data-title="Name:">Success</td>
     <td class="uqds-table__item" data-title="Variable:">Not yet defined</td>
+    <td class="uqds-table__item" data-title="HEX:">#</td>
     <td class="uqds-table__item" data-title="Usage:">Success, available</td>
   </tr>
   <tr class="uqds-table__row">
     <td class="uqds-table__item">Not yet defined</td>
     <td class="uqds-table__item" data-title="Name:">Warning</td>
     <td class="uqds-table__item" data-title="Variable:">Not yet defined</td>
+    <td class="uqds-table__item" data-title="HEX:">#</td>
     <td class="uqds-table__item" data-title="Usage:">Warning; there <em>may</em> be a problem</td>
   </tr>
   <tr  class="uqds-table__row">
     <td class="uqds-table__item"><div class="uqds-colour__palette uqds-red-dark20"></div></td>
     <td class="uqds-table__item" data-title="Name:">UQ Red Dark 40</td>
```

Comment:
Called "UQ Red Dark 40" yet the variable says "$uq-red-dark20"

Reply:
Fixed